knack/util.py

```diff
@@ -151,3 +151,18 @@ def todict(obj, post_processor=None):  # pylint: disable=too-many-return-stateme
                   if not callable(v) and not k.startswith('_')}
         return post_processor(obj, result) if post_processor else result
     return obj
+
+
+def isatty(stream):
+    # Code copied from
+    # https://github.com/tartley/colorama/blob/3d8d48a95de10be25b161c914f274ec6c41d3129/colorama/ansitowin32.py#L43-L53
+    import sys
+    if 'PYCHARM_HOSTED' in os.environ:
```

Comment:
PyCharm's integrated terminal is **not** a tty but it does support color, so detect if the command is run in PyCharm's integrated terminal. If so, enable color.

Reply:
Even though colorama already has this logic and can strip escape sequences `\033[` if the stream is not a TTY, we still need to detect it by ourselves so that we don't print escape sequences when colorama is not initialized. In other words, we shouldn't rely on colorama to do the stripping.

Reply:
Maybe you can put comment into code.